components/brave_new_tab_ui/containers/newTab/settings/backgroundImage.tsx

```diff
@@ -77,7 +76,7 @@ class BackgroundImageSettings extends React.PureComponent<Props, State> {
   }
 
   onClickBraveBackground = () => {
-    this.setState({ location: Location.BRAVE_BACKGROUNDS })
+    this.props.setBraveBackground('')
```

Comment:
Optional nit: I take it we can't pass `null` instead?

Reply:
for now the parameter is not optional. I think I can remove the param entirely in a follow-up.